Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -601,45 +565,20 @@ CGPathRef CGPathRetain(CGPathRef path) {
 }
 
 /**
- @Status Caveat
- @Notes transform property not supported
+@Status Stub
```

Comment:
What is our plan to track the apis that go from Interoperable/Caveat to Stubs?  We need to make these get back to those known states before merging to develop.